Lib/asyncio/base_events.py

```diff
@@ -806,7 +806,9 @@ def run_in_executor(self, executor, func, *args):
             # Only check when the default executor is being used
             self._check_default_executor()
             if executor is None:
-                executor = concurrent.futures.ThreadPoolExecutor()
+                executor = concurrent.futures.ThreadPoolExecutor(
+                    thread_name_prefix='asyncio_default_executor'
```

Comment:
A shorter name is better, please apply it.
```suggestion
                    thread_name_prefix='asyncio'
```